File: AIAP_LAB_18/car.py
'''Write a Python class Car with attributes: brand, model, year.
•	Car Details:
•	Brand: Toyota
•	Model: Corolla
•	Year: 2020
2.	Add a method display_details() that prints car details.
'''
class Car:
    def __init__(self, brand, model, year):
        self.brand = brand
        self.model = model
        self.year = year

    def display_details(self):
        print(f"Car Details:\nBrand: {self.brand}\nModel: {self.model}\nYear: {self.year}")
# Example usage
my_car = Car("Toyota", "Corolla", 2020)
my_car.display_details()

File: AIAP_LAB_18/Car.java
/* Translate the code present incar.py file into a Java class Car with attributes and a methoddisplayDetails(). */
public class Car {
    // Attributes
    private String make;
    private String model;
    private int year;

    // Constructor
    public Car(String make, String model, int year) {
        this.make = make;
        this.model = model;
        this.year = year;
    }

    // Method to display car details
    public void displayDetails() {
        System.out.println("Car Make: " + make);
        System.out.println("Car Model: " + model);
        System.out.println("Car Year: " + year);
    }

    // Main method for testing
    public static void main(String[] args) {
        Car myCar = new Car("Toyota", "Camry", 2020);
        myCar.displayDetails();
    }
}